e2e/detail.test.ts

```diff
@@ -0,0 +1,24 @@
+import { by, device, element, expect } from 'detox';
+
+describe('Detail', () => {
+    beforeAll(async () => {
+        await device.launchApp();
+    });
+
+    beforeEach(async () => {
+        await device.reloadReactNative();
+    });
+
+    it('should display detail screen', async () => {
+        await element(by.id('Menu')).tap();
+        await element(by.id('Detail')).tap();
+        await expect(element(by.id('Detail'))).toBeVisible();
+    });
+
+    it('should change index when pressing Next button', async () => {
```

Comment:
Add  another test case:
```ts
it(`should roll back to the last letter when pressing the Next button with the last letter selected`,...)
```
Here you should check that when you click all the way to 'z' (assuming you use fixture English alphabet data) and then click `Next` you return to `a`.

Note that you'll have a couple more cases to look at when you support `Prev`.